.github/dependabot.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+# Dependabot config that checks version updates for go.mod packages and docker
+# images, and also checks only for security updates for github actions.
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    # Disable version updates and only check security updates for github
+    # actions, since we can't bump the versions until they're on our allow-list
+    open-pull-requests-limit: 0
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Any reason not to match the cadence for the other 2?

Reply:
Good question. I borrowed this from VSO. I think your suggestion makes sense to match them. I would imagine that package updates are more frequent than docker image updates.

Reply:
Updated in bcde91bfd28ea5b0cad17012363b24cec6d13ed6

Reply:
(Just to add context here) IIRC the motivation for daily docker updates on VSO and vault-k8s was to catch ubi image updates, since RedHat has their own security scanning criteria that our release process wouldn't catch before pushing to RedHat's registry. CSI doesn't have a ubi image so 👍